components/sup/doc/http_gateway_services_schema.json

```diff
@@ -0,0 +1,521 @@
+            "description": "The name of this package"
+          },
+          "origin": {
+            "type": "string",
+            "description": "The origin of this package"
+          },
+          "path": {
+            "type": "string",
+            "description": "The path to the exploded version of this package on disk"
+          },
+          "release": {
+            "type": "string",
+            "description": "The release of this package"
+          },
+          "svc_config_path": {
+            "type": "string",
+            "description": "The path to this package's configuration",
+          },
+          "svc_data_path": {
+            "type": "string",
+            "description": "The path to this package's data",
+          },
+          "svc_files_path": {
+            "type": "string",
+            "description": "The path to this package's files",
+          },
+          "svc_group": {
+            "type": "string",
+            "description": "The group this package will run as",
+          },
+          "svc_path": {
+            "type": "string",
+            "description": "The root of this package's run time data",
+          },
+          "svc_pid_file": {
+            "type": "string",
+            "description": "The PID file for this package when it's running",
+          },
+          "svc_run": {
+            "type": "string",
+            "description": "The run hook",
+          },
+          "svc_static_path": {
+            "type": "string",
+            "description": "The path to this package's static files",
+          },
+          "svc_user": {
+            "type": "string",
+            "description": "The user this package will run as",
+          },
+          "svc_var_path": {
+            "type": "string",
+            "description": "The path to this package's var directory",
+          },
+          "version": {
+            "type": "string",
+            "description": "The version of this package",
+          }
+        }
+      },
+      "process": {
+        "type": "object",
+        "description": "Run time data about this service",
+        "required": [
+          "pid",
+          "state",
+          "state_entered"
+        ],
+        "properties": {
+          "pid": {
+            "type": "integer",
+            "description": "The PID of this process while it's running"
+          },
+          "state": {
+            "type": "string",
+            "description": "The state of this process, either Up or Down"
+          },
+          "state_entered": {
+            "type": "integer",
+            "description": "The time the process entered its current state"
+          }
+        }
+      },
+      "service_group": {
+        "type": "string",
+        "description": "The service group of this service"
+      },
+      "smoke_check": {
+        "type": "string",
+        "description": "The status of this service's smoke check, either Ok, Failed, or Pending"
```

Comment:
Enumerate values.

Though really we should make a note here that smoke checks aren't actually a thing... I think the main code for them got pulled a while ago, but it was an incomplete removal. 🙄